01/index.css

```diff
@@ -0,0 +1,9 @@
+/* NIE DZIAŁA OPŁYWANIE */
+* {
+    box-sizing: border-box;
+}
+
+.content::before {
+    float: left;
+    shape-outside: url(./assets/after.png);
```

Comment:
To jest obrazek w całości więc przeglądarka widzi po prostu prostokąt. Trzeba użyć `shape-outside: polygon`

Reply:
ok